MVVMIOS/Models/MovieResult.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  MovieResult.swift
+//  MVVMIOS
+//
+//  Created by Ravi Bastola on 3/6/22.
+//
+
+import Foundation
+
+struct MovieResult: Codable {
+    let id: Int
+    let name, coverImage, welcomeDescription, timePeriod: String
+    let references: String
+    let sortOrder: Int
+    let createdAt, updatedAt: String
+    let uuid: UUID = UUID()
```

Comment:
Unless you own this backend and can guarantee that these properties will never be nil I'd make these all optional. Otherwise your entire Codable will fail if even one item is ever nil.

Reply:
Okay Notes